src/AppPrintOnly.test.tsx

```diff
@@ -396,3 +397,106 @@ it('VxPrintOnly flow', async () => {
   // Default Unconfigured
   getByText('Device Not Configured')
 })
+
+test('VxPrint retains configuration after unload', async () => {
```

Comment:
let's not call it "configuration", it should only retain machine type and machine ID.

Reply:
Hardware configuration?

Reply:
Hardware state?

Reply:
How about "VxPrint retains app mode when unconfigured"?